triaging/index.rst

```diff
@@ -0,0 +1,13 @@
+===========================
+ Issue Tracking and Triage
+===========================
```

Comment:
```suggestion
===================
Issues and Triaging
===================
```

"Tracking" doesn't really add any informative value here, and triaging also includes PRs (which one of the two sections discusses, albeit with outdated terminology); otherwise, it would make more sense to call this subdirectory `issues` rather than triaging, as that would be the more general and appropriate term.